### ISSUE TITLE
Initial view too small to display contents

### ISSUE THREAD

**Issue by mauricefisher64** [ok-nick/c2pa-preview#7]
After dragging in asset the view is too small to show all the content.  The window is not resizable so you must scroll to see the rest of contents and inspect button.

**Comment by ok-nick** [comment]
Related #2

The issue has to do with `Window::setSize` in Tauri setting the entire window size, including the menu bar size, which isn't predictable because `Window::outerSize` and `Window::innerSize` incorrectly return the same values on macOS. I added a hacky workaround that calculates the difference between the HTML body size and Tauri window size on startup, then adds that to when the window size is being set.

TLDR; Should be fixed, I'm going to close this for now, let me know if you run into the same issue
